projects/packages/forms/src/blocks/dropzone/index.js

```diff
@@ -0,0 +1,108 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { lineDashed } from '@wordpress/icons';
+import { getIconColor } from '../contact-form/util/block-icons';
+import edit from './edit';
+import save from './save';
+
+const name = 'dropzone';
+const settings = {
+	apiVersion: 3,
+	title: __( 'File upload dropzone', 'jetpack-forms' ),
+	description: __( 'A dropzone for file uploads.', 'jetpack-forms' ),
+	parent: [ 'jetpack/field-file' ],
+	allowedBlocks: [
+		'core/buttons',
+		'core/heading',
+		'core/image',
+		'core/list',
+		'core/paragraph',
+		'core/separator',
+		'core/spacer',
+	],
+	category: 'contact-form',
+	icon: {
+		foreground: getIconColor(),
+		src: <Icon icon={ lineDashed } />,
+	},
+	attributes: {
+		style: {
+			type: 'object',
+			default: {
+				layout: {
+					type: 'constrained',
+					justifyContent: 'center',
+					orientation: 'vertical',
+				},
+				spacing: {
+					padding: {
+						top: '48px',
+						bottom: '48px',
+						left: '48px',
+						right: '48px',
+					},
+					margin: {
+						top: '8px',
+						bottom: '8px',
+					},
+				},
+				border: {
+					style: 'dashed',
+					width: '1px',
+					color: 'rgba(125,125,125,0.3)',
+				},
+			},
```

Comment:
Unfortunately, injecting these style values injects them at the individual block level which prevents global styles from working. Here's a screenshot of attempting to set an orange border within global styles, but the block level attributes are taking precedence:

<img width="1181" alt="image" src="https://github.com/user-attachments/assets/c03a0697-72d9-46b4-800a-be4d9fdf4c40" />

Could an alternative be to provide low-specificity CSS defaults of the styling that you want to as a baseline for this block, and then themes (and individual block markup) can override via the block supports if and when they need to?

This could be a neat alternative to the issue of picking the layout type, too, if there are still any outstanding issues with the layout type.